testkit/src/main/scala/org/bitcoins/testkit/core/gen/ScriptGenerators.scala

```diff
@@ -681,10 +848,20 @@ sealed abstract class ScriptGenerators extends BitcoinSLogger {
       keys: Seq[ECPublicKey]): LockTimeScriptSignature = {
 
     val nestedScriptSig = lock.nestedScriptPubKey match {
-      case _: P2PKScriptPubKey                                         => P2PKScriptSignature(sigs.head)
-      case _: P2PKHScriptPubKey                                        => P2PKHScriptSignature(sigs.head, keys.head)
-      case _: MultiSignatureScriptPubKey                               => MultiSignatureScriptSignature(sigs)
-      case EmptyScriptPubKey                                           => CSVScriptSignature(EmptyScriptSignature)
+      case _: P2PKScriptPubKey           => P2PKScriptSignature(sigs.head)
+      case _: P2PKHScriptPubKey          => P2PKHScriptSignature(sigs.head, keys.head)
+      case _: MultiSignatureScriptPubKey => MultiSignatureScriptSignature(sigs)
+      case conditional: ConditionalScriptPubKey =>
+        val spk = lock match {
+          case csv: CSVScriptPubKey =>
+            CSVScriptPubKey(csv.locktime, conditional.trueSPK)
+          case cltv: CLTVScriptPubKey =>
+            CLTVScriptPubKey(cltv.locktime, conditional.trueSPK)
+        }
+        val scriptSig = lockTimeHelperScriptSig(spk, sigs, keys).scriptSig
+        ConditionalScriptSignature(scriptSig, true)
+      case EmptyScriptPubKey =>
+        CSVScriptSignature(NonStandardScriptSignature("0151"))
```

Comment:
For the record, this was a bug that was never hit (since previously CLTV/CSV never wrapped `EmptyScriptPubKey`) where we must use `NonStandardScriptSignature(OP_TRUE)` rather than `EmptyScriptSignature` to satisfy an `EmptyScriptPubKey` in the `ScriptInterpreter`'s eyes

Reply:
Let's definitely add a comment in the source code as to what this is... `"0151"` is a magic value (I'm guessing it is 1 byte to push onto the stack and then OP_TRUE)